client/sites-dashboard/components/sites-grid-item.tsx

```diff
@@ -84,16 +86,19 @@ export const SitesGridItem = memo( ( { site }: SitesGridItemProps ) => {
 	return (
 		<SitesGridTile
 			leading={
-				<ThumbnailLink { ...siteDashboardUrlProps }>
-					<SiteItemThumbnail
-						displayMode="tile"
-						className={ siteThumbnail }
-						site={ site }
-						width={ THUMBNAIL_DIMENSION.width }
-						height={ THUMBNAIL_DIMENSION.height }
-						sizesAttr={ SIZES_ATTR }
-					/>
-				</ThumbnailLink>
+				<>
+					<ThumbnailLink { ...siteDashboardUrlProps }>
+						<SiteItemThumbnail
+							displayMode="tile"
+							className={ siteThumbnail }
+							site={ site }
+							width={ THUMBNAIL_DIMENSION.width }
+							height={ THUMBNAIL_DIMENSION.height }
+							sizesAttr={ SIZES_ATTR }
+						/>
+					</ThumbnailLink>
+					site.plan?.expired && <SitesGridActionRenew site={ site } />
```

Comment:
We need to wrap this line with curly braces as this is an expression
```suggestion
					{ site.plan?.expired && <SitesGridActionRenew site={ site } /> }
```

![image](https://user-images.githubusercontent.com/26530524/197836805-442a3f88-b2f9-443f-af6a-0a40d76aab04.png)

😬